solarforecastarbiter/reference_forecasts/tests/test_forecast.py

```diff
@@ -38,18 +27,31 @@ def test_resample():
     assert forecast.resample(None) is None
 
 
-def test_interpolate():
-    index = pd.date_range(start='20190101', freq='15min', periods=2)
-    arg = pd.Series([0, 1.5], index=index)
-    out = forecast.interpolate(arg)
-    assert_series_equal(out, arg)
+@pytest.fixture
+def rfs_series():
+    return pd.Series(
+        [1, 2], index=pd.DatetimeIndex(['20190101 01', '20190101 02']))
```

Comment:
can we have another that has a nan in there somewhere

Reply:
what about all nan? what about just ``pd.Series()``

Reply:
not sure why the function would get data like [1, 2, nan, 4] but can document the behavior

all nan works, but will add test. 

what should the behavior be for empty and None? 

```
    if arg is None:
        return None
    if arg.empty:
        return arg
```

?

Reply:
sure

Reply:
added tests